README.md

```diff
@@ -37,4 +39,9 @@ with an `(I)` appended e.g. `EMIS (I)`. The addition of `(I)` represents a GP
 that is now using the Informatica system.
 
 Note: The list above was created by running
-`jq -c '[.[].Supplier] | unique ' output/pomi.json`
+`jq -c '[.[].Supplier] | unique ' html/json/pomi.json`
+
+The json files created will be available via the container's webserver at
```

Comment:
Summary of discussion: the missing file was caused by a network error. At the moment all three files are downloaded concurrently which is fine for 3 files, but may need to move to a sequential approach if more file are added.

Created an issue to add retries on network timeouts https://github.com/nhsuk/pomi-data-etl/issues/18